### PR TITLE
20487D | Mod06 | Demo

### PR DESCRIPTION
**Pull request by VinayApposite**
CE complete. Comments inline.

### CODE REVIEW

File: Instructions/20487D_MOD06_DEMO.md [hunk 1]
```diff
@@ -1,163 +1,162 @@
...
     ```bash
     sudo chown myadmin demo
     ```
-26. Open new **Command Line** instance.
-27. Paste the following command to change directory to the published folder:
+26. Open a new command prompt instance.
+27. To change the directory to the published folder, paste the following command:
     ```bash
-    [Repository Root]\Allfiles\Mod06\Demofiles\Demo2Project\bin\release\netcoreapp2.1
+    *[Repository Root]*\Allfiles\Mod06\Demofiles\Demo2Project\bin\release\netcoreapp2.1
```
Comment (VinayApposite) [inline Instructions/20487D_MOD06_DEMO.md]: Should there be a "cd" here?

File: Instructions/20487D_MOD06_DEMO.md [hunk 2]
```diff
@@ -1,163 +1,162 @@
-# Module 6: Deploying and managing services
+# Module 6: Deploying and Managing Services
```
Comment (VinayApposite) [inline Instructions/20487D_MOD06_DEMO.md]: There are two "Lesson 3" and "Lesson 5". Is that correct?

File: Instructions/20487D_MOD06_DEMO.md (hunk 1 above)
Comment (slavBarkal) [reply Instructions/20487D_MOD06_DEMO.md]: yes

File: Instructions/20487D_MOD06_DEMO.md (hunk 2 above)
Comment (slavBarkal) [reply Instructions/20487D_MOD06_DEMO.md]: no, there should be two lesson 5, 
Lesson 3: Web Deployment on Linux - should be lesson 2

**Comment by AviAvni** [comment]
@VinayApposite Is this finished?

**Comment by VinayApposite** [comment]
Yes, this is complete.

**Comment by barucht** [comment]
Ready for QA

**Pull request merged**